File: python_backend/game_logic/color/get_ai_color.py


def get_ai_color(player_type):
    """ """
    if player_type == "test":
        return None
    if player_type == "W":
        return "B"
    if player_type == "B":
        return "W"


if __name__ == "__main__":
    get_ai_color("test")

File: app (1)/src/game_logic/color/getAiColor.js
export function getAiColor(playerType) {
    /* **/
    if (playerType === "test") {
        return undefined
    }
    if (playerType === "W") {
        return "B"
    }
    if (playerType === "B") {
        return "W"
    }
}

// module.exports = getAiColor;

/** for node.js
if (require.main === module) {
    getAiColor("test")

}
*/